src/boot/compat.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line
+import { configureCompat } from 'vue'
+
+// TODO delete file
```

Comment:
I _think_ this is just because of croppa...

```suggestion
// for compatibility with vue-croppa
// can be deleted once vue-croppa supports vue 3 or we don't use it anymore
// see https://github.com/zhanziyang/vue-croppa/issues/235
```